client/death/FormPage.js

```diff
@@ -238,6 +256,7 @@ export default class IndexPage extends React.Component {
 
             <div className="txt">
               <label htmlFor="text" className="txt-l">Dog Age (Years)</label>
+              <style textAlign="right" />
```

Comment:
What are you trying to achieve here? I think you probably want `style={{textAlign: 'right'}}` on the `<input />` below.